doc/source/changes/version_0_31.rst.inc

```diff
@@ -3,16 +3,9 @@
 New features
 ^^^^^^^^^^^^
 
-*
-
-
-.. _misc_editor:
-
-Miscellaneous improvements
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-*
-
+* implemented run_editor_on_exception function. If you call this function in your code (for example at the top of
```

Comment:
:py:obj:`run_editor_on_exception()` ?

Reply:
Unsure it will work given we are not within larray module and there is no api doc for larray_editor itself. If it works, of course that's better...

Reply:
If the docstring is moved to corresponding wrapper function the larray project and considering that the content of the present file will be merged in the corresponding in the larray project, it should work right?